src/services/production.ts

```diff
@@ -239,8 +239,11 @@ export const services: ServiceConfigurator<SupportedTokens> = ($) => [
       $.store,
     ],
   }],
+
+  // Modules
+  ...zoneModule($),
   ...meshes($),
-  ...servicesModule($),
+  ...serviceModule($),
```

Comment:
Could you put this back please?

They are all currently plural, like the folder names, I would also prefer it if you used `zones($)` like the rest.

Reply:
I’ve made them both use the plural.